tests/unit/amo/components/TestAddonHead.js

```diff
@@ -276,4 +277,23 @@ describe(__filename, () => {
 
     expect(root.find('link[rel="alternate"]')).toHaveLength(0);
   });
+
+  // This test case ensures the production configuration is taken into account.
+  it.each([['x-default', 'en-US'], ['pt', 'pt-PT'], ['en', 'en-US']])(
+    'renders a "x-default" alternate link',
```

Comment:
Nit: Should this description use one of the variables from the `it.each` instead of always saying `"x-default"`?

Reply:
oops